tests/loadtests/glloadtests/shader-based/BasisuTest.cpp

```diff
@@ -108,7 +108,7 @@ BasisuTest::BasisuTest(uint32_t width, uint32_t height,
     else if (features.etc1)
         tf = KTX_TF_ETC1;
     else if (features.pvrtc1)
-        tf = KTX_TF_PVRTC1_4_OPAQUE_ONLY;
+        tf = KTX_TF_BC7_M6_RGB;
```

Comment:
This is not correct. features.pvrtc1 will only ever be true if PVRTC1 support is detected and BC7 has nothing to do with PVRTC1. Further detection logic is needed in GL3LoadTestSample to detect BC7 whose result should be checked here. Please build on what I did in PR #131.